aws-mfa

```diff
@@ -273,8 +297,8 @@ def get_credentials(short_term_name, lt_key_id, lt_access_key, args, config):
     with open(AWS_CREDS_PATH, 'w') as configfile:
         config.write(configfile)
     logger.info(
-        "Success! Your credentials will expire in %s seconds at: %s"
-        % (args.duration, response['Credentials']['Expiration']))
+        "Success! Your credentials will expire at: %s" %
+        response['Credentials']['Expiration'])
```

Comment:
`args.duration` will still have a value when this is called, so it should still log.